client/app/js/css/style.css

```diff
@@ -95,6 +95,12 @@ input {
     width: 300px;
     margin: 150px auto 0px auto;
 }
+#messageboard {
+    height:120px;
+    width: 500px;
+    margin-top: 50px;
+    margin-left: 40px;
```

Comment:
[np] order your style properties alphabetically. So we should have height, margin-left, margin-top, width